pkg/testcase/clusterreset.go

```diff
@@ -0,0 +1,91 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestClusterReset() {
+	cluster := factory.ClusterConfig(GinkgoT())
+
+	killall(cluster)
+	shared.LogLevel("INFO", "%s-service killed", cluster.Config.Product)
+
+	stopServer(cluster)
+	shared.LogLevel("INFO", "%s-service stopped", cluster.Config.Product)
```

Comment:
I think I mentioned it in some other PR, we need to provide the correct string format in the `shared.LogLevel` func as it expects the log level as lowercase `info` but here we are passing `INFO`. So I am not sure if the case with get hit as the func expectation is different: https://github.com/rancher/distros-test-framework/blob/main/shared/aux.go#L335